server/sonar-server/src/main/java/org/sonar/server/platform/monitoring/CeTasksMonitor.java

```diff
@@ -39,9 +41,12 @@ public String name() {
   }
 
   @Override
-  public LinkedHashMap<String, Object> attributes() {
+  public Optional<Map<String, Object>> attributes() {
     try (JmxConnection connection = jmxConnectionFactory.create(ProcessId.COMPUTE_ENGINE)) {
-      LinkedHashMap<String, Object> result = new LinkedHashMap<>();
+      if (connection == null) {
+        return Optional.absent();
+      }
+      Map<String, Object> result = new LinkedHashMap<>();
```

Comment:
implementation details checked, the actual backing array size is the nearest superior power of two, so here, 8. We gain 50% usage.